presto-main/src/main/java/com/facebook/presto/operator/PartitionedOutputOperator.java

```diff
@@ -220,7 +220,8 @@ public PartitionedOutputOperator(
                 serdeFactory,
                 sourceTypes,
                 maxMemory,
-                operatorContext.getDriverContext().getLifespan());
+                operatorContext.getDriverContext().getLifespan(),
```

Comment:
No need to pass this anymore. It can be acquired from operatorContext passed in.